gk/main.c

```diff
@@ -512,6 +586,9 @@ gk_proc(void *arg)
 		struct rte_mbuf *rx_bufs[GATEKEEPER_MAX_PKT_BURST];
 		struct rte_mbuf *tx_bufs[GATEKEEPER_MAX_PKT_BURST];
 		struct gk_cmd_entry *gk_cmds[GK_CMD_BURST_SIZE];
+		struct acl_search acl = {
+			.num = 0,
+		};
```

Comment:
Create a macro for this definition and initialization. Motivation: being the less intrusive on the blocks that help the IPv6 filter.